index.html

```diff
@@ -14,26 +14,164 @@
          
     </nav>
 
+    <section class="landing-page">
+    <div class="all-body" id="body-cover">
+        
+            <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <p class="intro">
+                I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't 
+                hestiate to contact me.
+            </p>
+        
+            <p class="lets-connect">LET'S CONNECT</p>
+                
+            <ul class="social-buttons">
+                <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
+            </ul>
+        
+    </div>
+    <section>
+        
+    <section class="works-container">
+        <div class="works">
+            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
+            <h1 class="header-tonic">Tonic</h1>
 
+            <ul class="description">
+                <li class="canopy">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                <li class="back">2015</li>
+            </ul>
 
-    <section class="all-body" id="body-cover">
-        <h2 class="title">I'm Mariane Glad to see you!</h2>
-        <p class="intro">
-            I'm a software developer! I can help you build a product , 
-            feature or website Look through some of my work and experience! 
-            If you like what you see and have a project you need coded, don't 
-            hestiate to contact me.
-        </p>
-    
-        <p class="lets-connect">LET'S CONNECT</p>
+            <p class="more-info">A daily selection of privately 
+                personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li><button class="lang" id="lan-1">html</button></li>
+                <li><button class="lang" id="lan-2">css</button></li>
+                <li><button class="lang" id="lan-3">javaScript</li></button></li>
+            </ul>
+
+            <button class="see" id="see-1">See Project</button>
```

Comment:
[OPTIONAL] You have followed the recommendation of adding button interaction, by forgot to implement the color scheme given in the Figma design share by previous reviewer.
![image](https://user-images.githubusercontent.com/61361037/221237996-e6e313ca-1b57-4c54-9270-078edafbbfd9.png)

Kindly follow this.
![image](https://user-images.githubusercontent.com/61361037/221237683-5b9ca6a4-a9b7-4087-a61f-7f5162c7a78b.png)